specs/spec_helper.rb

```diff
@@ -0,0 +1,12 @@
+require "simplecov"
+SimpleCov.start
+
+require "minitest"
+require "minitest/spec"
+require "minitest/autorun"
+require "minitest/reporters"
+
+require_relative "../far_mar"
```

Comment:
Because you have this line in your `spec_helper`, you won't need it in any of the spec files.